packages/inventory/src/components/table/helpers.test.js

```diff
@@ -42,15 +42,11 @@ describe('createRows', () => {
     const cells = [{ key: 'first' }, { key: 'second.dot' }, { key: 'third' }];
     it('should create empty table', () => {
         const data = createRows();
-        const wrapper = mount(data[0].cells[0].title);
```

Comment:
The component is now outside the helpers, the tests still pass and the actual rendering of the `NoSystemsResults` component is tested in the `InventoryTable` tests.